api/v1/latency.go

```diff
@@ -0,0 +1,71 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/celestiaorg/bittwister/xdp/latency"
+	"go.uber.org/zap"
+)
+
+// PacketlossStart implements POST /packetloss/start
+func (a *RESTApiV1) LatencyStart(resp http.ResponseWriter, req *http.Request) {
+	var body LatencyStartRequest
+	if err := json.NewDecoder(req.Body).Decode(&body); err != nil {
+		sendJSONError(resp,
+			MetaMessage{
+				Type:    APIMetaMessageTypeError,
+				Slug:    SlugJSONDecodeFailed,
+				Title:   "JSON decode failed",
+				Message: err.Error(),
+			},
+			http.StatusBadRequest)
+		return
+	}
+
+	if a.lt == nil {
+		a.lt = &netRestrictService{
+			service: &latency.Latency{
+				Latency: time.Duration(body.Latency) * time.Millisecond,
+				Jitter:  time.Duration(body.Jitter) * time.Millisecond,
+			},
+			logger: a.logger,
+		}
+	} else {
+		lt, ok := a.lt.service.(*latency.Latency)
+		if !ok {
+			sendJSONError(resp,
+				MetaMessage{
+					Type:    APIMetaMessageTypeError,
+					Slug:    SlugTypeError,
+					Title:   "Type cast error",
+					Message: "could not cast netRestrictService.service to *packetloss.PacketLoss",
```

Comment:
PacketLoss in error message of latency method

Reply:
ty fixed